Sources/NIOHTTP2/ConnectionStateMachine/ConnectionStateMachine.swift

```diff
@@ -1753,20 +1753,27 @@ extension HTTP2StreamID {
     }
 }
 
+extension HTTP2ConnectionStateMachine.ConnectionRole {
+    /// Obtain the inverse role to self
+    /// - Returns: The inverse role
+    fileprivate func inverse() -> Self {
+        switch self {
+        case .server:
+            return .client
+        case .client:
+            return .server
+        }
+    }
+}
```

Comment:
This refactor can probably be backed out: we aren't calling this anywhere other than where it was originally defined, so out-of-lining it hasn't brought us any utility. Let's remove the churn in the code instead.